src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/Validation/ClientModelValidationContext.cs

```diff
@@ -22,20 +27,14 @@ public class ClientModelValidationContext
                 throw new ArgumentNullException(nameof(metadataProvider));
             }
 
-            if (requestServices == null)
-            {
-                throw new ArgumentNullException(nameof(requestServices));
-            }
-
             ModelMetadata = metadata;
             MetadataProvider = metadataProvider;
-            RequestServices = requestServices;
         }
 
+        public ActionContext ActionContext { get; }
```

Comment:
We had services here in lieu of the `ActionContext` in the past. Now that this is here it seems silly to keep services as a top-level property.